app/javascript/components/patient/laboratory/LaboratoryModal.js

```diff
@@ -52,8 +52,8 @@ class LaboratoryModal extends React.Component {
   };
 
   render() {
-    // Data is valid and can be saved if there is either a report or specimen collection date AND a result
-    const isValid = (this.state.report || this.state.specimen_collection) && this.state.result;
+    // Data is valid as long as at least one field has a value
```

Comment:
Since a date field is no longer required, thoughts on adding an isClearable flag to those inputs to make it easier to remove?

Reply:
same thought on allowing clearing of lab test type

Reply:
Good point, all fields are now clearable in https://github.com/SaraAlert/SaraAlert/pull/1086/commits/bffd465e574dccd8a08d49794f68ef0fb32906d9.